.eslintrc.js

```diff
@@ -1,72 +1,6 @@
 module.exports = {
-  globals: {
-    React: true,
-    google: true,
```

Comment:
Most controversial: I loosened the linting rules, found it to painful to work with them.